themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -50,7 +50,7 @@
 
     {block name='product_unit_price'}
       {if $displayUnitPrice}
-        <p class="product-unit-price">{convertPrice price=$unit_price} {l s='per %s' sprintf=$product.unity}</p>
+        <p class="product-unit-price"{hook h='displayProductPriceBlock' product=$product type="unit_price" returntype="style"}>{convertPrice price=$unit_price} {l s='per %s' sprintf=$product.unity}</p>
```

Comment:
Thanks for your feedback,

To avoid using this hook directly in the HTML tag, we could create a new ```front.js``` file that would be included thanks to the same hook call we can see on line 54 (right after this one).

And this ```front.js``` file would contains something like this:

```javascript
$(document).ready(function(){
    $('p.product-unit-price').css('display', 'none');
});
````

What do you think ?

Best regards,